AWSPluginsCore.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   AMPLIFY_VERSION = '1.0.1'
-  AWS_SDK_VERSION = "~> 2.13.4"
+  AWS_SDK_VERSION = '~> 2.13.4'
```

Comment:
Can we take this opportunity to fix my mistake of including the "compatibility squiggle" in the version declaration as well? That is, let's make both version declarations be just bare semvers, and decorate them as needed (with `"~>"` compatibility prefix, or the `"v"` prefix) at the call sites.